WhaleSpotting/react-app/src/styles/Profile.scss

```diff
@@ -0,0 +1,133 @@
+@import './Constants.scss';
+
+.body {
+    max-width: 1024px;
+    font-family: $fontFamily;
+    margin: 0 auto 0 auto;
+    display: flex;
+    flex-direction: row;
+}
+
+.profile-pane {
+    height: 100vh;
```

Comment:
When we add a header and footer in this may become an issue, also when we add in content, I would suggest saying `max-height: calc(100vh - $footerHeight - $headerHeight)